java/HealthVault Mobile/HVSample/src/main/java/com/microsoft/hsg/android/hvsample/RecordPickerArrayAdapter.java

```diff
@@ -49,14 +49,30 @@ public long getItemId(int position) {
     }
 
     @Override
-    public View getView(int position, View view, ViewGroup parent) {
-        View rowView= layoutInflater.inflate(R.layout.record_picker_item, null, true);
-        TextView txtTitle = (TextView) rowView.findViewById(R.id.txtrecordName);
-        ImageView imageView = (ImageView) rowView.findViewById(R.id.recordIcon);
-        txtTitle.setText(records.get(position).getName());
-        
-        imageLoader.load(records.get(position).getId(), imageView, R.drawable.ic_launcher);
-        
-        return rowView;
+    public View getView(int position, View convertView, ViewGroup parent) {
+        ViewHolderItem viewHolder;
+        if(convertView == null) {
+            viewHolder = new ViewHolderItem();
+            convertView = mLayoutInflater.inflate(R.layout.record_picker_item, null, true);
+
+            viewHolder.textViewItem = (TextView) convertView.findViewById(R.id.txtrecordName);
```

Comment:
We are still using a mix of naming conventions for resources. The id's here are somewhat camelCase, above the layout is underscore separated and other places in the review we do the same. we should be consistent.